src/app/src/components/EmbeddedMapFieldsConfig.jsx

```diff
@@ -271,10 +331,24 @@ function EmbeddedMapFieldsConfig({
                                             />
                                         }
                                         label="Select All"
+                                        style={{ width: '312px' }}
```

Comment:
Changing this to `284px` will make the "Searchable" heading appear (approximately) centered above the checkboxes.